.github/workflows/trigger-release-candidate-build-and-deploy.yml

```diff
@@ -3,16 +3,16 @@ name: Trigger release candidate build and deploy
 on:
   push:
     branches:
-      - main
+      - psridharan/trigger-stage-deploy # main
 
 jobs:
-  merge:
+  continuous_deploy_to_staging:
```

Comment:
If decide to _not_ run unit tests on staging branch (redundantly with main), then we will need to check that unit tests passed on main before proceeding.